### PR TITLE
fixed parameter file

### PR DESCRIPTION
**Pull request by xtophs**
names to match azuredeploy.json
values to match types defined in azuredeploy.json

### CODE REVIEW

**Review by Smarker — APPROVED** [review]
LGTM

File: azuredeploy.parameters.json [hunk 1]
```diff
@@ -6,13 +6,13 @@
...
-            "value": "1"
+            "value": 1
         },
         "acsAgentCount": {
-            "value": "6"
+            "value": 6
         },
         "agentVMSize": {
-            "value": "Standard_D2_v2"
+            "value": "Standard_DS2"
```
Comment (erikschlegel) [inline azuredeploy.parameters.json:15]: Can you please change the `agentVMSize` to `Standard_L8s`? Most of our clusters are now running on L-series vms.

**Review by erikschlegel — APPROVED** [review]
LGTM, with just one minor comment.

**Review by jcjimenez — APPROVED** [review]
LGTM

File: azuredeploy.parameters.json (hunk 1 above)
Comment (xtophs) [reply azuredeploy.parameters.json:15]: L-series is not available in all data centers yet. Even in the datacenters where L-series is available it doesn't seem to have all sizes.

For example for eastus, I get: 

The requested VM size Standard_L4s is not available in the current region. The sizes available in the current region are: Standard_DS1_v2,Standard_DS2_v2,Standard_DS3_v2,Standard_DS4_v2,Standard_DS5_v2,Standard_DS11_v2,Standard_DS12_v2,Standard_DS13_v2,Standard_DS14_v2,Standard_DS15_v2,Standard_DS2_v2_Promo,Standard_DS3_v2_Promo,Standard_DS4_v2_Promo,Standard_DS5_v2_Promo,Standard_DS11_v2_Promo,Standard_DS12_v2_Promo,Standard_DS13_v2_Promo,Standard_DS14_v2_Promo,Standard_F1s,Standard_F2s,Standard_F4s,Standard_F8s,Standard_F16s,Standard_A0,Standard_A1,Standard_A2,Standard_A3,Standard_A5,Standard_A4,Standard_A6,Standard_A7,Basic_A0,Basic_A1,Basic_A2,Basic_A3,Basic_A4,Standard_D1_v2,Standard_D2_v2,Standard_D3_v2,Standard_D4_v2,Standard_D5_v2,Standard_D11_v2,Standard_D12_v2,Standard_D13_v2,Standard_D14_v2,Standard_D15_v2,Standard_D2_v2_Promo,Standard_D3_v2_Promo,Standard_D4_v2_Promo,Standard_D5_v2_Promo,Standard_D11_v2_Promo,Standard_D12_v2_Promo,Standard_D13_v2_Promo,Standard_D14_v2_Promo,Standard_F1,Standard_F2,Standard_F4,Standard_F8,Standard_F16,Standard_A1_v2,Standard_A2m_v2,Standard_A2_v2,Standard_A4m_v2,Standard_A4_v2,Standard_A8m_v2,Standard_A8_v2,Standard_H8,Standard_H16,Standard_H8m,Standard_H16m,Standard_H16r,Standard_H16mr,Standard_D1,Standard_D2,Standard_D3,Standard_D4,Standard_D11,Standard_D12,Standard_D13,Standard_D14,Standard_NV6,Standard_NV12,Standard_NV24,Standard_DS1,Standard_DS2,Standard_DS3,Standard_DS4,Standard_DS11,Standard_DS12,Standard_DS13,Standard_DS14,Standard_NC6,Standard_NC12,Standard_NC24,Standard_NC24r,Standard_A8,Standard_A9,Standard_A10,Standard_A11,Standard_D2_v3,Standard_D4_v3,Standard_D8_v3,Standard_D16_v3,Standard_D2s_v3,Standard_D4s_v3,Standard_D8s_v3,Standard_D16s_v3,Standard_E2_v3,Standard_E4_v3,Standard_E8_v3,Standard_E16_v3,Standard_E32_v3,Standard_E64_v3,Standard_E2s_v3,Standard_E4s_v3,Standard_E8s_v3,Standard_E16s_v3,Standard_E32s_v3,Standard_E64s_v3. Find out more on the available VM sizes in each region at https://aka.ms/azure-regions.

Even in westus where L-series is available, I get: 
The VM Size of Master, Agent is not allowed in your subscription in location 'westus'. Master VM Size 'Standard_D2_v2' is available in locations: eastus, southcentralus, westeurope, southeastasia. Agent VM Size 'Standard_L8s' is available in locations: westeurope, southeastasia.

How about we stick with default values that work out of the box in all datacenters until L-series is widely available?L-series is not available in all data centers yet. Even in the datacenters where L-series is available it doesn't seem to have all sizes.

For example for eastus, I get: 

The requested VM size Standard_L4s is not available in the current region. The sizes available in the current region are: Standard_DS1_v2,Standard_DS2_v2,Standard_DS3_v2,Standard_DS4_v2,Standard_DS5_v2,Standard_DS11_v2,Standard_DS12_v2,Standard_DS13_v2,Standard_DS14_v2,Standard_DS15_v2,Standard_DS2_v2_Promo,Standard_DS3_v2_Promo,Standard_DS4_v2_Promo,Standard_DS5_v2_Promo,Standard_DS11_v2_Promo,Standard_DS12_v2_Promo,Standard_DS13_v2_Promo,Standard_DS14_v2_Promo,Standard_F1s,Standard_F2s,Standard_F4s,Standard_F8s,Standard_F16s,Standard_A0,Standard_A1,Standard_A2,Standard_A3,Standard_A5,Standard_A4,Standard_A6,Standard_A7,Basic_A0,Basic_A1,Basic_A2,Basic_A3,Basic_A4,Standard_D1_v2,Standard_D2_v2,Standard_D3_v2,Standard_D4_v2,Standard_D5_v2,Standard_D11_v2,Standard_D12_v2,Standard_D13_v2,Standard_D14_v2,Standard_D15_v2,Standard_D2_v2_Promo,Standard_D3_v2_Promo,Standard_D4_v2_Promo,Standard_D5_v2_Promo,Standard_D11_v2_Promo,Standard_D12_v2_Promo,Standard_D13_v2_Promo,Standard_D14_v2_Promo,Standard_F1,Standard_F2,Standard_F4,Standard_F8,Standard_F16,Standard_A1_v2,Standard_A2m_v2,Standard_A2_v2,Standard_A4m_v2,Standard_A4_v2,Standard_A8m_v2,Standard_A8_v2,Standard_H8,Standard_H16,Standard_H8m,Standard_H16m,Standard_H16r,Standard_H16mr,Standard_D1,Standard_D2,Standard_D3,Standard_D4,Standard_D11,Standard_D12,Standard_D13,Standard_D14,Standard_NV6,Standard_NV12,Standard_NV24,Standard_DS1,Standard_DS2,Standard_DS3,Standard_DS4,Standard_DS11,Standard_DS12,Standard_DS13,Standard_DS14,Standard_NC6,Standard_NC12,Standard_NC24,Standard_NC24r,Standard_A8,Standard_A9,Standard_A10,Standard_A11,Standard_D2_v3,Standard_D4_v3,Standard_D8_v3,Standard_D16_v3,Standard_D2s_v3,Standard_D4s_v3,Standard_D8s_v3,Standard_D16s_v3,Standard_E2_v3,Standard_E4_v3,Standard_E8_v3,Standard_E16_v3,Standard_E32_v3,Standard_E64_v3,Standard_E2s_v3,Standard_E4s_v3,Standard_E8s_v3,Standard_E16s_v3,Standard_E32s_v3,Standard_E64s_v3. Find out more on the available VM sizes in each region at https://aka.ms/azure-regions.

Even in westus where L-series is available, provisioning Standard_L8s I get: 
The VM Size of Master, Agent is not allowed in your subscription in location 'westus'. Master VM Size 'Standard_D2_v2' is available in locations: eastus, southcentralus, westeurope, southeastasia. Agent VM Size 'Standard_L8s' is available in locations: westeurope, southeastasia.

Same error for Standard_L4s:
The VM Size of Master, Agent is not allowed in your subscription in location 'westus'. Master VM Size 'Standard_D2_v2' is available in locations: eastus, southcentralus, westeurope, southeastasia. Agent VM Size 'Standard_L4s' is available in locations: westeurope, southeastasia.

Thoughts about keeping default values that work out of the box in all datacenters until L-series is widely available?

**Pull request merged**